### DISCUSSION TITLE
Allow multiple rest time intervals

### DISCUSSION THREAD

**Discussion by hubmacfan** [astashov/liftosaur#150] [Ideas]
Currently, the user can only set a timer that apply to all  warm up sets and  another timer for all working sets. I suggest allowing the user to specify rest time for each exercise in a routine. For example, in the Blacknoir GZCLP, you have different rest times between sets of your T1 lifts than you do between sets of the T2 or T3 lifts.

**Comment by yrinkel** [comment]
For the working sets at least, you can specify custom rest timers with the ```120s``` syntax (you can check the doc for more details):
https://www.liftosaur.com/blog/docs/

**Comment by hubmacfan** [comment]
I was wondering if I could. I haven't been brave enough to try
creating/editing in Liftoscript yet.

**Comment by astashov** [comment]
Yeah, that exists already, both through UI in the app, and through the program text.
